modules/@apostrophecms/schema/ui/apos/components/AposInputBoolean.vue

```diff
@@ -177,4 +179,30 @@ export default {
     border-color: var(--a-base-2);
     box-shadow: 0 0 5px var(--a-base-6);
   }
+
+  input[disabled],
+  input[disabled]:checked {
+    & + label,
```

Comment:
I see we already used the tag a few times for styling, but we should probably get back to styling with classes.